README.md

```diff
@@ -1,26 +1,102 @@
-# gatsby-starter-default
-The default Gatsby starter.
+<p align="center">
+  <a href="https://next.gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Gatsby's default starter
+</h1>
 
-For an overview of the project structure please refer to the [Gatsby documentation - Building with Components](https://www.gatsbyjs.org/docs/building-with-components/).
+Kick off your your project with this default boilerplate. This barebones starter ships with the main Gatsby configuration files you might need. 
 
-## Install
+_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://next.gatsbyjs.org/docs/gatsby-starters/)._
 
-Make sure that you have the Gatsby CLI program installed:
-```sh
-npm install --global gatsby-cli
-```
+## 🚀 Quick start
 
-And run from your CLI:
-```sh
-gatsby new gatsby-example-site
-```
+1.  **Install the Gatsby CLI.**
 
-Then you can run it by:
-```sh
-cd gatsby-example-site
-gatsby develop
-```
+    The Gatsby CLI helps you create new sites using Gatsby starters (like this one!)
 
-## Deploy
+    ```sh
+    # install the Gatsby CLI globally
+    npm install -g gatsby-cli
+    ```
+
+2.  **Create a Gatsby site.**
+
+    Use the Gatsby CLI to create a new site, specifying the default starter.
+
+    ```sh
+    # create a new Gatsby site using the default starter
+    gatsby new my-default-starter https://github.com/gatsbyjs/gatsby-starter-default#v2
```

Comment:
Should this point to `v2` branch? Gatsby v2 is still in beta.

`master` branch here uses Gatsby v1. `v2` branch here uses Gatsby v2 beta.

Perhaps we should remove `#v2` part here, and after this is merge cherry-pick this to `v2` and then use `#v2` part?

Reply:
Good catch! I'll make this one the simple `gatsby new my-default-starter`, and add the `#v2` part to the `v2` branch